command-line.txt

```diff
@@ -1,11 +1,11 @@
 What is the command that displays your current working directory?
-your_answer = ____________
+your_answer = pwd
 
 What is the command to list all of the files and directories in your current working directory?
-your_answer = ____________
+your_answer = ls command-line-proficiency-2015
```

Comment:
WHOA! I didn't know you could do that. Thanks for teaching me that. Assume you are in the repository directory anyway. How would your answer change?